CONTRIBUTING.md

```diff
@@ -27,8 +27,15 @@ Fork using GitHub Desktop:
 - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
 
 Fork using the command line:
+
 - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
 
+Fork, edit and preview using [GitHub Codespaces](https://github.com/features/codespaces):
+
+- With one click of a button, simply fork, start to edit and preview the documentation, without having to install anything.
+
+  ![](./assets/images/codespaces.gif)
```

Comment:
Thanks for creating this gif! However, since we don't include any gifs for the other instructions, it breaks the guide up a bit. Can we leave it out for now and re-assess adding visuals for contributing later?

Reply:
Sure. Let me remove it.

Reply:
Done.